app/assets/stylesheets/common/components/date-input.scss

```diff
@@ -0,0 +1,13 @@
+.d-date-input {
+  .date-picker {
+    margin: 0;
+    text-align: left;
+    width: 100%;
+    outline: none;
+    box-shadow: none !important;
```

Comment:
:cry:

Reply:
yes agreed, my feeling is that I will be able to remove old pickers in the future and then remove this, so Im more eager to accept it 🤷‍♂️